### PR TITLE
Add documentation key as discussed in #2549

### PR DESCRIPTION
**Pull request by jspricke**

### CODE REVIEW

File: .travis.sh [hunk 1]
```diff
@@ -99,14 +99,8 @@ function doc ()
...
 
-  if [ -z "$id_rsa_{1..23}" ]; then echo 'No $id_rsa_{1..23} found !' ; exit 1; fi
-
-  echo -n $id_rsa_{1..23} >> ~/.ssh/travis_rsa_64
-  base64 --decode --ignore-garbage ~/.ssh/travis_rsa_64 > ~/.ssh/id_rsa
-
-  chmod 600 ~/.ssh/id_rsa
-
   echo -e "Host github.com\n\tStrictHostKeyChecking no\n" >> ~/.ssh/config
+  mv .travis.key "$HOME/.ssh/id_ed25519"
```
Comment (taketwo) [inline .travis.sh]: What about merging this into the `openssl` command that happens in `before_install` step? That is, decrypt directly into `$HOME/.ssh/id_ed25519`.

Comment (jspricke) [reply .travis.sh]: Tried that here: https://travis-ci.com/PointCloudLibrary/pcl/jobs/151599120

**Comment by taketwo** [comment]
More importantly though is that build failed with:

```
$ openssl aes-256-cbc -K $encrypted_275e3339731d_key -iv $encrypted_275e3339731d_iv -in .travis.key.enc -out .travis.key -d
iv undefined
The command "openssl aes-256-cbc -K $encrypted_275e3339731d_key -iv $encrypted_275e3339731d_iv -in .travis.key.enc -out .travis.key -d" failed and exited with 1 during .
Your build has been stopped.
```

There are a few googlable threads that discuss this, but no clear solution. How did you login before encrypting, with `--com` or with `--org`?

File: .travis.sh (hunk 1 above)
Comment (taketwo) [reply .travis.sh]: I see. Looks like shell variable expansion does not work.

**Comment by jspricke** [comment]
Yeah, no idea why. I used `--com`

File: .travis.sh (hunk 1 above)
Comment (jspricke) [reply .travis.sh]: We can try the other way round and move the `openssl`  here.

**Comment by taketwo** [comment]
Obsoleted by #2604.

**Pull request closed without merging**